js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx

```diff
@@ -25,129 +31,203 @@ const ViewSettingsReactView = ({ name }) => {
-                <th className="collectionTh">
-                  Type:
-                </th>
-                <th className="collectionTh">
-                  <div className="modal-text" id="change-view-type">
-                    {formState.type}
-                  </div>
-                </th>
-              </tr>
-              </tbody>
-            </table>
+  return (
+    <div className={"centralContent"} id={"content"}>
+      <div
+        id={"modal-dialog"}
+        className={"createModalDialog"}
+        tabIndex={-1}
+        role={"dialog"}
+        aria-labelledby={"myModalLabel"}
+        aria-hidden={"true"}
+      >
+        <div className="modal-body">
+          <div className={"tab-content"}>
+            <div className="tab-pane tab-pane-modal active" id="General">
+              <table style={{ margin: 30, marginLeft: 20 }}>
+                <tbody>
+                  <tr className="tableRow" id="row_change-view-name">
+                    <th className="collectionTh">Name*:</th>
+                    <th className="collectionTh">
+                      <Textbox
+                        type={"text"}
+                        value={formState.name}
+                        onChange={updateName}
+                        required={true}
+                        disabled={frontendConfig.isCluster || !isAdminUser}
+                      />
+                    </th>
+                    <th className="collectionTh">
+                      <ToolTip
+                        title="The View name (string, immutable)."
+                        setArrow={true}
+                      >
+                        <span className="arangoicon icon_arangodb_info"></span>
+                      </ToolTip>
+                    </th>
+                  </tr>
+
+                  <tr
+                    className="tableRow"
+                    id="row_change-view-cleanupIntervalStep"
+                  >
+                    <th className="collectionTh">Cleanup Interval Step:</th>
+                    <th className="collectionTh">
+                      <Textbox
+                        type={"number"}
+                        disabled={!isAdminUser}
+                        value={getNumericFieldValue(
+                          formState.cleanupIntervalStep
+                        )}
+                        onChange={getNumericFieldSetter(
+                          "cleanupIntervalStep",
+                          dispatch
+                        )}
+                      />
+                    </th>
+                    <th className="collectionTh">
+                      <ToolTip
+                        title="The steps to wait before removing unused segments after release of internal resource."
+                        setArrow={true}
+                      >
+                        <span className="arangoicon icon_arangodb_info"></span>
+                      </ToolTip>
```

Comment:
A bit of new code that was almost missed.

Reply:
Done.